base/arrayshow.jl

```diff
@@ -102,7 +102,7 @@ function print_matrix_row(io::IO,
         if isassigned(X,Int(i),Int(j)) # isassigned accepts only `Int` indices
             x = X[i,j]
             a = alignment(io, x)
-            sx = sprint(0, show, x, env=io)
+            sx = sprint(show, x, context=io, sizehint=0)
```

Comment:
Since `sizehint=0` is the default, the keyword could have been omitted here...  presumably, it was only being passed before because that was previously the only way to pass `env`.